tests/behaviour/connection/connection_steps.py

```diff
@@ -32,3 +33,8 @@ def step_impl(context: Context):
 @step("connection does not have any database")
 def step_impl(context: Context):
     assert len(context.client.databases().all()) == 0
+
+
+@step("wait {seconds} seconds")
+def step_impl(context: Context, seconds: str):
```

Comment:
We should create `util_steps.py` and move this step there, as we did in Client Java